### ISSUE TITLE
Modes of transportation options

### ISSUE THREAD

**Issue by jdupuis** [ProteanSoftware/capacitor-start-navigation#10]
Hello !

Is it possible to add in the StartNavigationOptions a mode of transportation ?

e.g: I prefer to launch directly the navigation app in cycling mode instead of driving mode.

Thanks a lot !

**Comment by AshleyMedway** [comment]
This should be available in version 4.1.0.

there is a new `travelMode` option, values
```ts
export type TravelMode = 'driving' | 'walking' | 'transit' | 'bicycling';
```

Defaults to 'driving'.
All options are available on web.
On iOS only 'driving', 'walking' & 'transit' are available.
On Android only 'driving', 'walking', 'bicycling' are available.

**Comment by jdupuis** [comment]
Already done ! Thank you very much @AshleyMedway !

Version 4.1.0 is already released, so I'm going to test it.

Thank you again for this quick improvement! I'll come back to you if there are any problems.

**Comment by jdupuis** [comment]
Hi @AshleyMedway ,

I'll come back to you after my tests on this improvement.

Regarding the ios version, I have nothing to say, it works well even if it's a shame that we can't (yet?) request the "bicycling" mode but I understand that it's an Apple Plans limitations.

On the other hand, for the Android version, it doesn't work as expected. The navigation mode is added in the destination field (mode=d, see attached capture) and therefore we do not switch to the desired navigation mode. Also, I feel like the destination mode is not retrieved correctly because mode is always equal to 'd', whether travelMode = 'walking' or 'bycicling' in my calls to StartNavigation.launchMapsApp()..

Is it possible to correct this?

Thanks in advance !

![Screenshot_2023-02-22-12-17-33-621_com google android apps maps](https://user-images.githubusercontent.com/1327450/221142419-b5db72bd-edd2-49d8-99b9-3eed303e688a.jpg)

**Comment by AshleyMedway** [comment]
@jdupuis Yes, with Apple Maps they do not have an an option to start with cycling unfortunately.

Google maps should be fixed in ~~4.1.1~~ 4.1.3

**Comment by jdupuis** [comment]
ok @AshleyMedway , it's works well now in 4.1.3!

Thanks a lot !
